dti-website/src/data/all-projects.json

```diff
@@ -187,61 +48,24 @@
-        "image": "/static/projects/queuemein-feature2.jpg"
-      },
-      {
-        "title": "View the Queue.",
-        "description": "You can find an opportunity to listen in for a question you also have, or view other questions asked to understand overall problem areas.",
-        "image": "/static/projects/queuemein-feature3.jpg"
-      }
-    ]
-  },
-  {
-    "name": "Research Connect",
-    "active": false,
-    "teamId": "research",
-    "card": "/static/projects/researchconnect-card.png",
-    "header": "Research Connect",
-    "subheader": "Handshake for Lab Research.",
-    "hero": {
-      "header": "What We Do",
-      "image": "",
-      "subheader": "Large universities have so many research opportunities, but few ways to find out about them. Research Connect is a platform where students can find opportunities and research labs can find students they’re looking for."
-    },
-    "website": "https://www.research-connect.com",
-    "website-title": "Research Connect",
-    "github": "https://github.com/cornell-dti/research-connect",
-    "external": "http://cornellsun.com/2018/04/24/cornellians-share-business-plans-in-competition-promoting-entrepreneurship/",
-    "heroStartingColor": "#FAACA8",
-    "heroEndingColor": "#FAACA8",
-    "heroUseDarkText": false,
+    "heroStartingColor": "#569ADB",
+    "heroEndingColor": "#D3E8FD",
+    "heroUseDarkText": true,
     "features": [
       {
-        "title": "Find Research Opportunities.",
-        "description": "Instead of having to email researchers, we streamline the process by showing undergrads all of the opportunities available to them.",
-        "image": "/static/projects/researchconnect-feature1.jpg"
+        "title": "Queue",
+        "description": "The queue feature allows students to post their question and TAs to assign themselves to a student in an orderly fashion.",
+        "image": "/static/projects/queuemein-feature1.png"
       },
       {
-        "title": "View Requirements. Apply Online.",
-        "description": "Find out exactly what requirements a lab is searching for that match your background.",
-        "image": "/static/projects/researchconnect-feature2.jpg"
+        "title": "Discussion",
+        "description": "The discussion feature allows students to vote on a question so that the TAs can address the most popular/urgent question.",
+        "image": "/static/projects/queuemein-feature2.png"
       },
       {
-        "title": "Search Faculty.",
-        "description": "Search for faculty across all departments to find out who is conducting research and whether they’re looking for students. (Coming Soon)",
-        "image": "/static/projects/researchconnect-feature3.jpg"
+        "title": "Dashboard/Analytics",
+        "description": "The analytics feature allows TAs and professors to view statistics about each tags or assignments such as total number of questions asked and the percentage of questions answered.",
```

Comment:
Small typo here as well: "tags" and "assignments" should be switched to "tag" and "assignment" or it could be "all tags and assignments".